.gitignore

```diff
@@ -1,4 +1,5 @@
 .DS_Store
+package-lock.json
```

Comment:
The package-lock.json shouldn't be in the `.gitignore`, as we need to have it in the repo
Instead you can fix the commit which edits `package-lock.json` using an interactive rebase
Or I can do it for you if you prefer?

Reply:
Uhhhh, I think you better do that :), I don't think I can figure that out without messing something up!